src/components/Login/Login.js

```diff
@@ -71,6 +71,9 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
         setErrors({ _general: 'validation_messages.error_unexpected' });
       } else if (result.expectedError && result.expectedError.accountNotValidated) {
         setErrors({ _general: <LoginErrorAccountNotValidated email={values[fieldNames.user]} /> });
+      } else if (result.expectedError && result.expectedError.accountIsCanceled) {
+        // TODO: define content when the account is canceled
```

Comment:
Could you validate it with @carodipietro before merging?

Reply:
Oh, this content already exists in Doppler, could you use Doppler content by default? (maybe you are already doing that, in that case, please remove the TODO comment)

Reply:
No, I invented this comment but for now I will use it the same that in Doppler. I'd leave this TODO comment to remember change it when Carolina has the real content.

Reply:
Este contenido no está generado para esta versión. Podés usar lo que está en Doppler actualmente hasta que se generen los nuevos contenidos.